state_machine/plugs/flight/lib/pycubed.py

```diff
@@ -147,289 +147,467 @@ def __init__(self):
+            burnwires.append(self.burnwire2)
+        except Exception as e:
+            print('[ERROR][Burn Wire IC1]', e)
+
+        burnwire_count = 0
+        for burnwire in burnwires:
+            burnwire_count += 1
+        
+        if burnwire_count >= 1:
+            self.hardware['BurnWire'] = True
+        
 
+    # reinit: reinitialize radio, sd, or IMU based upon the contents of string dev
     def reinit(self, dev):
-        dev = dev.lower()
+        # dev is a whitespace-stripped string of all lowercase letters
+        dev = dev.strip().lower()
+
+        # reinitialize device based on string dev
         if dev == 'radio':
-            self.radio2.__init__(self.spi, self._rf_cs, self._rf_rst, self.UHF_FREQ)
+            # should we be reinitializing radio2 or just radio?
+            self.radio.__init__(self.spi, self._rf_cs, self._rf_rst, self.UHF_FREQ)
         elif dev == 'sd':
             self._sd.__init__(self.spi, self._sdcs, baudrate=1000000)
         elif dev == 'imu':
             self.IMU.__init__(self.i2c1)
         else:
-            print('Invalid Device? ->', dev)
+            print('Invalid Device? ->',dev)
+
 
     @property
+    # return the accelerometer reading from the IMU
     def acceleration(self):
         return self.IMU.accel
 
+
     @property
+    # return the magnetometer reading from the IMU
     def magnetic(self):
         return self.IMU.mag
 
+
     @property
+    # return the gyroscope reading from the IMU
     def gyro(self):
         return self.IMU.gyro
 
+
     @property
+    # return the thermometer reading from the IMU
     def temperature(self):
-        return self.IMU.temperature  # Celsius
+        return self.IMU.temperature # Celsius
+
 
     @property
+    # return the temperature reading from the CPU
     def temperature_cpu(self):
-        return microcontroller.cpu.temperature  # Celsius
+        return microcontroller.cpu.temperature # Celsius
+
 
     @property
+    # return the current RBG settings of the neopixel object
     def RGB(self):
         return self.neopixel[0]
 
+    
     @RGB.setter
-    def RGB(self, value):
+    # set an RGB value to the neopixel object
+    def RGB(self,value):
         if self.hardware['Neopixel']:
             try:
                 self.neopixel[0] = value
             except Exception as e:
-                print('[WARNING]', e)
+                print('[WARNING]',e)
+
 
     @property
+    # return the battery voltage
     def battery_voltage(self):
-        vbat = 0
+        # initialize vbat
+        vbat=0
+
         for _ in range(50):
-            vbat += self._vbatt.value * 3.3 / 65536
-        _voltage = (vbat / 50) * (100 + 100) / 100  # 100k/100k voltage divider
-        return _voltage  # volts
+            # 65536 = 2^16, number of increments we can have to voltage
+            vbat+=self._vbatt.value * 3.3 / 65536
+        
+        # 316/110 voltage divider
```

Comment:
These are the incorrect resistor values, stick to the updated values from here:
```python
_voltage = (vbat / 50) * (100 + 100) / 100  # 100k/100k voltage divider
```

Reply:
good catch